application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksSystem.java

```diff
@@ -0,0 +1,186 @@
+
+        return isOverviewChannelName.test(parentChannelName);
+    }
+
+    boolean didUserBookmarkChannel(long userID, long channelID) {
+        return database.read(context -> context.selectFrom(BOOKMARKS)
+            .where(BOOKMARKS.AUTHOR_ID.eq(userID), BOOKMARKS.CHANNEL_ID.eq(channelID))
+            .fetch()
+            .isNotEmpty());
+    }
+
+    void addBookmark(long authorID, long channelID, @Nullable String note) {
+        database.write(context -> context.newRecord(BOOKMARKS)
+            .setAuthorId(authorID)
+            .setChannelId(channelID)
+            .setCreatedAt(Instant.now())
+            .setNote(note)
+            .setScheduledRemovalAt(null)
+            .insert());
+    }
+
+    List<BookmarksRecord> getUsersBookmarks(long authorID) {
+        return database.read(context -> context.selectFrom(BOOKMARKS)
+            .where(BOOKMARKS.AUTHOR_ID.eq(authorID))
+            .orderBy(BOOKMARKS.CREATED_AT.desc())
+            .fetch());
+    }
+
+    void removeBookmarks(long authorID, List<Long> channelIDs) {
+        database.write(context -> context.deleteFrom(BOOKMARKS)
+            .where(BOOKMARKS.AUTHOR_ID.eq(authorID), BOOKMARKS.CHANNEL_ID.in(channelIDs))
+            .execute());
+    }
+
+    int getTotalBookmarkCount() {
+        return database
+            .read(context -> context.selectCount().from(BOOKMARKS).fetchOne(0, int.class));
+    }
+
+    int getUserBookmarkCount(long authorID) {
+        return database.read(context -> context.selectCount()
+            .from(BOOKMARKS)
+            .where(BOOKMARKS.AUTHOR_ID.eq(authorID))
+            .fetchOne(0, int.class));
+    }
+
+    void scheduleUsersBookmarksRemoval(long authorID) {
+        Instant scheduledRemovalAt = Instant.now().plus(LEAVE_REMOVAL_DELAY);
+
+        database.write(context -> context.update(BOOKMARKS)
+            .set(BOOKMARKS.SCHEDULED_REMOVAL_AT, scheduledRemovalAt)
+            .where(BOOKMARKS.AUTHOR_ID.eq(authorID))
+            .execute());
+    }
+
+    void cancelUsersBookmarksRemoval(long authorID) {
+        database.write(context -> context.update(BOOKMARKS)
+            .setNull(BOOKMARKS.SCHEDULED_REMOVAL_AT)
+            .where(BOOKMARKS.AUTHOR_ID.eq(authorID))
+            .execute());
+    }
+
+    void cleanRemovalScheduledBookmarks() {
+        database.write(context -> context.deleteFrom(BOOKMARKS)
+            .where(BOOKMARKS.SCHEDULED_REMOVAL_AT.isNotNull(),
+                    BOOKMARKS.SCHEDULED_REMOVAL_AT.lessThan(Instant.now()))
+            .execute());
+    }
+
+    /**
+     * Creates an embed with just a description and color.
+     *
+     * @param description The embed description
+     * @param color The embed color
+     * @return The generated embed
+     */
+    static MessageEmbed createTinyEmbed(String description, Color color) {
+        return new EmbedBuilder().setDescription(description).setColor(color).build();
+    }
```

Comment:
this method is just calling a few other methods, i'd say remove this method, i can see this method being helpful if u have a fixed color but u r passing the color too

compare these
```java
BookmarkSystem.createTinyEmbed(description, color);
```
```java
new EmbedBuilder().setDescription(description).setColor(color).build();
```

if anything second one is more clear on what it is doing

Reply:
The color is changing for warnings and errors

Reply:
This moving it to a new method was actually suggested by someone else and i agree with it. It gets very messy at the top of a class if you create a new EMbedBuilder every time.

Reply:
> The color is changing for warnings and errors

i'd say instead make 2 methods createWarningEmbed and createErrorEmbed and both will take only a description

Reply:
I like this idea!. Adding the color makes it look bad anyways

Reply:
![image](https://user-images.githubusercontent.com/54677650/198838816-f0dcb42c-7cd4-4244-9c74-e834af56517f.png)
 On windows they are pretty much just a line... We should also go with emojis for your navigation buttons.